### PR TITLE
Fix useWindowsDriver when other opengl driver are used like intel-compute-runtime

### PR DESCRIPTION
**Pull request by SuperSandro2000**
see https://github.com/nix-community/NixOS-WSL/pull/339#discussion_r1441907386

### CODE REVIEW

File: modules/wsl-distro.nix [hunk 1]
```diff
@@ -76,8 +76,11 @@ in
 
       extraPackages = mkIf cfg.useWindowsDriver [
         (pkgs.runCommand "wsl-lib" { } ''
-          mkdir "$out"
-          ln -s /usr/lib/wsl/lib "$out/lib"
+          mkdir -p "$out/lib"
+          # we cannot just symlink the lib directory because it breaks merging with other drivers that provide the same directory
+          for so in libcuda.so libd3d12.so libd3d12core.so libdxcore.so; do
```
Comment (SuperSandro2000) [inline modules/wsl-distro.nix:81]: Do we need other cuda so's? What are the amd names?

Comment (Atry) [reply modules/wsl-distro.nix:81]: AMD does not support CUDA

Comment (SuperSandro2000) [reply modules/wsl-distro.nix:81]: I meant other shared objects in general, like the amd one. What's the name for it? Is there even one?

**Comment by SuperSandro2000** [comment]
see #383

**Pull request closed without merging**